libs/sdk-backend-bear/src/convertors/dateFormatting/defaultDateFormatter.ts

```diff
@@ -0,0 +1,21 @@
+// (C) 2020 GoodData Corporation
+import format from "date-fns/format";
+import { UnexpectedError } from "@gooddata/sdk-backend-spi";
+
+import { DateFormatter } from "./types";
+import { dateFormats, DEFAULT_DATE_FORMAT } from "./dateValueParser";
+
+/**
+ * Creates a default date formatting function.
+ * @public
+ */
+export const createDefaultDateFormatter = (dateFormat: string = DEFAULT_DATE_FORMAT): DateFormatter => {
+    return (value: Date, targetDateFormat: string = dateFormat): string => {
+        if (!dateFormats.includes(targetDateFormat)) {
+            throw new UnexpectedError(
+                `Unsupported date format "${targetDateFormat}". Supported date formats are ${dateFormats}`,
+            );
+        }
+        return format(value, targetDateFormat);
```

Comment:
de we care about the locale?
I saw this on tiger https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-backend-tiger/src/convertors/fromBackend/dateFormatting/defaultDateFormatter.ts#L64

Reply:
I think we don't care about the locale in this feature. Also it seems that there is no use case requiring a locale to format a date in Bear (if there is such a case, it would be added to the codebase at the same time as the code for Tiger you mentioned above).

Reply:
ok